test/libsolidity/smtCheckerTests/inheritance/constructor_state_variable_init_diamond_middle.sol

```diff
@@ -0,0 +1,22 @@
+pragma experimental SMTChecker;
+
+contract A {
+	uint x = 1;
+}
+
+contract B is A {
+	uint x = 2;
```

Comment:
I guess these will be shadowing errors in the future anyways... having ``y`` here and ``z`` in ``C``, depending on each other, would go towards my other comments about test cases...

Reply:
But depending on each other how? The contracts are unrelated